packages/components/src/components/swipeable-wrapper/swipeable-wrapper.tsx

```diff
@@ -8,6 +8,7 @@ type TSwipeableWrapper = {
     onChange?: (prop?: number) => void;
     is_disabled?: boolean;
     is_swipe_disabled?: boolean;
+    should_lift_up_navigation?: boolean;
```

Comment:
```suggestion
    should_elevate_navigation?: boolean;
```

Reply:
Sorry, can we change it to `elevate` across the code for better readability? Having extra underscore/hyphen is quite disturbing here.

Reply:
Sure, sorry